nextjs-13/src/workers/sendMail.ts

```diff
@@ -0,0 +1,52 @@
+import Handlebars from 'handlebars';
+import { createTransport } from 'nodemailer';
+
+import { queryNewsletters } from 'repositories/newsletter.repository';
+
+const sendMail = async (job) => {
+  const { senderId, senderName, ids, to } = job.data;
+
+  const newsletters = await queryNewsletters(senderId, ids);
+  const items = newsletters.map((newsletter) => {
+    return {
+      id: newsletter.id,
+      name: newsletter.name,
+    };
+  });
+
+  const source =
+    '<p>Nimble Newsletter just invited you to view these newsletters:</p>' +
+    '<ul>' +
+    '{{#each items}}' +
+    `<li><a href="${process.env.NEXTAUTH_URL}/newsletter/{{this.id}}">{{this.name}}</a></li>` +
+    '{{/each}}' +
+    '</ul>';
+
+  const template = Handlebars.compile(source);
+  const html = template({ items });
+
+  const transporter = createTransport({
+    host: process.env.MAILGUN_SMTP_HOST,
+    port: process.env.MAILGUN_SMTP_PORT,
+    // change host and port envs to this to use maildev mail catcher
+    // host: '0.0.0.0',
+    // port: 1025,
```

Comment:
These shouldbe define in the `.env` file instead. So mail catcher for local development, and nailgun for production/

Reply:
Yes that's what I mean here too, it's just that I don't know where to put this note into 👀 Maybe putting them in the README is a better idea.

Reply:
You can add them in https://github.com/nimblehq/js-framework-benchmark/pull/121/files#diff-795b86af04c107fdf257bbc42f5724fea71ba2b6e7cfd39ca8df44415df5e68b (env.sample)

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/121/files#diff-795b86af04c107fdf257bbc42f5724fea71ba2b6e7cfd39ca8df44415df5e68bR8-R14)